src/Settings/WalletSettings/WalletSettingsScreen.tsx

```diff
@@ -89,7 +85,7 @@ export const WalletSettingsScreen = () => {
 
       <SettingsSection>
         <NavigatedSettingsItem label={strings.removeWallet} navigateTo="remove-wallet" />
-        <ResyncButton />
+        <ResyncButton wallet={wallet} />
```

Comment:
imo -> move the confirmation dialog to `onClick` and if the users confirm fire the hook, is it a button or is it a menu item? Anyway I'd expect to have `onClick` API here.